lib/registration.pm

```diff
@@ -115,12 +115,18 @@ sub fill_in_registration_data {
                 @scc_addons = grep { $_ ne 'ltss' } @scc_addons;
             }
             for my $addon (@scc_addons) {
-                if (check_var('VIDEOMODE', 'text') || check_var('SCC_REGISTER', 'console')) {
+                if (check_var('VIDEOMODE', 'text') || check_var('SCC_REGISTER', 'console') || check_var('ZDUP', '1')) {
```

Comment:
Better use `get_var('ZDUP')` instead of `check_var('ZDUP', '1')` to check boolean values